wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,331 @@
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 2011-03-20 | 2011-04-23 | [\[1\]](https://web.archive.org/web/20130905210807/http://puu.sh/1gIC) |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 2011-04-23 | 2011-05-18 | [\[1\]](https://web.archive.org/web/20110423224541/http://osu.ppy.sh/p/playerranking) |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-05-18 | 2011-06-05 | [\[1\]](https://web.archive.org/web/20110518091909/http://osu.ppy.sh/p/playerranking/?m=0&s=2&r=&q=&f=) |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 2011-06-05 | 2011-06-16 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-06-16 | 2011-11-05 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2011-11-05 | *2012-04-20* |  |
+
+## 2012
+
+*Note: ppv1 fully replaced Ranked Score on 24 July 2012.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2011-11-05* | 2012-04-20 |  |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | 2012-04-20 | 2012-04-21 | [\[1\]](https://web.archive.org/web/20210104072637/http://geckogates.pristonworld.com/osu/gz_num1/) [\[2\]](https://osu.ppy.sh/community/forums/topics/77231?start=1474307) |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2012-04-21 | 2012-04-29 | [\[1\]](https://web.archive.org/web/20120421025437/http://osu.ppy.sh/p/playerranking) |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | 2012-04-29 | 2012-05-01 | [\[1\]](https://web.archive.org/web/20120429092504/http://osu.ppy.sh/p/playerranking) |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2012-05-01 | 2012-11-30 | [\[1\]](https://web.archive.org/web/20120503012726/http://osu.ppy.sh/u/39828)  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2012-11-30 | *2013-01-16* |  |
+
+## 2013
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | *2012-11-30* | 2013-01-16 |  |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 2013-01-16 | 2013-01-21 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2013-01-21 | 2013-10-04 |  |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 2013-10-04 | 2013-10-05 |  |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 2013-10-05 | 2013-10-20 |  |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 2013-10-20 | 2013-12-29 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2013-12-29 | *2014-01-27* |  |
+
+## 2014
+
+*Note: ppv2 was introduced on 27 January 2014.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2013-12-29* | 2014-01-27 |  |
```

Comment:
While gathering sources for the start of rrtyui's reign, I restumbled upon this news post: https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance

While I still need to gather sources for the leaderboard before this date, I want to see what people's opinions are on leaving 2014-01-20 to 2014-01-27 as vacant as PP was completely disabled for this period and when reenabled there was a new system and new rank 1. The three options are:

Option 1: Leave as is

```
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2013-12-29* | 2014-01-27 |  |
```

Option 2: Only change WubWoofWolf To date

```suggestion
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2013-12-29* | 2014-01-20 |  |
```

Option 3: Changing WubWoofWolf To date and adding a Vacant to the list.

```suggestion
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2013-12-29* | 2014-01-20 |  |
| **Vacant** | 2014-01-20* | 2014-01-27 | [\[1\]](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)  |
```

Reply:
I lean to either option 1 or 3. Option 2 is just confusing.

Reply:
Personally, I lean towards Option 3, as it's more straightforward and raises less eyebrows

Reply:
Going to go with option 3, cheers for the feedback.